includes/html/table/alertlog.inc.php

```diff
@@ -13,15 +13,16 @@
  * @author     LibreNMS Contributors
 */
 
-$alert_severities = [
+$alert_severities = array(
```

Comment:
please change "array()" back to "[.,..]"
"array()" is old style

Reply:
done